codecov.yaml

```diff
@@ -0,0 +1,193 @@
+codecov:
+  require_ci_to_pass: true
+
+coverage:
+  precision: 2
+  round: down
+  range: 70...100
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: true
+      loop: true
+      method: false
+      macro: false
+
+comment:
+  layout: reach,diff,flags,files,footer
+  behavior: default
+  require_changes: false
+
+flag_management:
+  default_rules:
+    carryforward: true
+
+ignore:
+  - "**/test"
+
+fixes:
+  - "autoware.universe/*/accel_brake_map_calibrator/::"
```

Comment:
Why do you have to include `autoware.universe` here?
I mean, usually this kind of settings are written as the paths from the repository root and don't have their repository name.